content/homeserver/room_directory.md

```diff
@@ -3,14 +3,42 @@ title = "Matrix.org Room Directory"
+
+1. The matrix.org public room directory is highly visible and serves as the main entrypoint into the Matrix ecosystem for many new users. To ensure trust and safety on matrix.org and maintain a good onboarding experience, the Matrix foundation curates the room directory.
+2. Rooms or spaces may be included in the room directory if they meet the following criteria.
+   1. Rooms must be actively moderated. There must be active users with a moderator role, and spam and abuse should be removed within a reasonable time period.
+   2. Room moderators must be reasonably responsive to foundation staff in the case of questions or concerns.
+   3. Rooms must comply with best practices for moderation of large public rooms, as determined by the Trust and Safety Committee of the Governing Board.
+   4. Rooms named after a recognizable organization or project must be sponsored by that project or be clearly marked as unofficial, while still having a positive relationship with the project.
+   5. In the case of a space, all rooms under the space must comply with this policy, and the administrators of the space must provide assurance that this will continue to be the case.
+3. Approval of rooms for inclusion in the room directory is at the discretion of the Room Directory Working Group. Generally, the Working Group expects to approve rooms that are related to Matrix, other open source projects or communities, or other well-defined topics that are of interest to the Matrix community. Because of the difficulty of reviewing and moderating “general” rooms with no specific focus, they will not generally be approved.
+   1. The Room Directory Working Group will consult Foundation Trust & Safety staff prior to approving additions. In some cases, Trust & Safety staff may object to the addition of a room.
+4. Administrators of rooms aligned with this policy may request that they be added to the room directory by sending an email to *TBD* with a general description of the room’s topic and its moderation practices. The working group may follow up with additional questions.
+5. Matrix foundation staff may remove rooms from the room directory if they are found to be in violation of the code of conduct or this policy, or if removal is required to mitigate an ongoing incident.
+6. The Room Directory Working Group will review the directory contents at least annually to determine if rooms should remain listed.
+
+**Interpretation notes**
+
+* Clause 4 supports using forms instead of an email address.
+* Clause 2's use of "may" permits approval *and* denial.
+* Clause 3 is permissive to "vaguely related to Matrix", but is intended to be kept relatively narrow to "Matrix-related" for now. This may change in the future.
```

Comment:
```suggestion

* Clause 2's use of "may" permits approval *and* denial.
* Clause 3 is permissive to "vaguely related to Matrix", but is intended to be kept relatively narrow to "Matrix-related" for now. This may change in the future.
* Clause 4 supports using forms instead of an email address.
```

Feels weird to have them out of order, and I hope/assume fixing the order doesn't involve a bunch of bureaucracy :D